### ISSUE TITLE
VictoriaLogs VMUI: setting changed after executing query

### ISSUE THREAD

**Issue by Nello-Angelo** [VictoriaMetrics/VictoriaMetrics#7371]
### Describe the bug

In the front, I set the search parameters I need - which fields to group by, time, etc... after that, in the very back right corner in the settings, I click “apply”, then I click “execute query” and all the settings disappear, but if I Before clicking “execute query”, I reboot the page and the settings are saved.

### To Reproduce

In the front, I set the search parameters I need - which fields to group by, time, etc... after that, in the very back right corner in the settings, I click “apply”, then I click “execute query” and all the settings disappear, but if I Before clicking “execute query”, I reboot the page and the settings are saved.

### Version

chart 0.7.0

### Logs

-

**Comment by jiekun** [comment]
@Nello-Angelo Hi. I just tried to reproduce it, but I didn't see any issues. Could you provide a screen recording of the steps you took and the bug you observed?

**Comment by Nello-Angelo** [comment]
[Видеовстреча 2024-10-27_182052.webm](https://github.com/user-attachments/assets/a951093a-3fa6-476e-b53d-8d637823a7df)

**Comment by Nello-Angelo** [comment]
[Uploading Видеовстреча 2024-10-27_182650.webm…]()

**Comment by Nello-Angelo** [comment]
can you watch?

**Comment by jiekun** [comment]
> can you watch?

Nope. It seems the link is pointing to the issue itself.

**Comment by Nello-Angelo** [comment]
[Видеовстреча 2024-10-27_182650.webm](https://github.com/user-attachments/assets/d9376960-8497-4f6e-9d38-a3f4d42001ca)

**Comment by Nello-Angelo** [comment]
i can send in telegram group english

**Comment by Nello-Angelo** [comment]
or lets call in discord

**Comment by Nello-Angelo** [comment]
i can upload correct

**Comment by tenmozes** [comment]
i got the file from @Nello-Angelo and let me upload it

[2024-10-27_183958.webm](https://github.com/user-attachments/assets/90d60abf-ecc3-4401-9482-d17c38e601c5)

**Comment by jiekun** [comment]
Thanks @tenmozes and @Nello-Angelo! I've labeled this as VL and VMUI's bug. Maybe @Loori-R could take a look later.

**Comment by hagen1778** [comment]
It is likely a duplicate of https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7279 and was fixed in https://github.com/VictoriaMetrics/VictoriaMetrics/pull/7294. But not released yet, see https://docs.victoriametrics.com/victorialogs/changelog/#tip

@Nello-Angelo can you try building vmlogs from source to verify that issue still exists? Let me know if you want me to build an image for you.

**Comment by jiekun** [comment]
I can do it for you. Just a sec.

**Comment by jiekun** [comment]
@Nello-Angelo sorry for the late. Could you try the following image:
```
victoriametrics/victoria-logs:heads-master-0-g5d73b8b86-dirty-3e2d3dc3-amd64
```

**Comment by Nello-Angelo** [comment]
yes no problem

**Comment by Nello-Angelo** [comment]
yes the problem is resolved

**Comment by hagen1778** [comment]
Thanks for feedback @Nello-Angelo ! The fix was merged and will be available in the next vlogs release.

@jiekun I've returned `bug` label because this ticket describes different problem than listed in https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7279. So let's count it as a new bug report.

**Comment by valyala** [comment]
The bug should be fixed in [v0.38.0-victorialogs](https://docs.victoriametrics.com/victorialogs/changelog/). Closing the issue as fixed.
